### ISSUE TITLE
scatter3() and plot3() do not work as expected according to labels

### ISSUE THREAD

**Issue by t37w** [jheinen/GR.jl#109]
Calling scatter3() or plot3() to create a 3D graphics it plots data, axes, labels (zlabel looks a little misplaced) but emits also a warning in the REPL of julia-0.6.2 on Windows.
     Invalid keyword: zlabel
Thus I grepped thru the the src file jlgr.jl.  It mentions 'zlabel' only once.
And 'zlabel' is not part of the array named 'kw_args'.
A small example:
using GR
x=rand(101); y=rand(101); z=rand(101);
scatter3(x,y,z,xlabel="x",ylabel="y", zlabel="z")

Best Regards,
Thomas

**Comment by jheinen** [comment]
Thanks for the hint. Fixed in 1be3268
